packages/ui/cypress/tests/address-bar.cy.ts

```diff
@@ -263,6 +263,9 @@ describe('Account address in the address bar', () => {
 
     topMenuItems.desktopMenu().within(() => topMenuItems.networkSelector().click())
     topMenuItems.networkSelectorOption('kusama').click()
+    landingPage
```

Comment:
@asnaith this actually made it so that it'd have failed earlier, I tested by removing my fix. It's better than having to wait 2s on purpose.

Reply:
Yep makes sense :)